viewing_party/party.py

```diff
@@ -1,23 +1,217 @@
+import statistics 
+from statistics import mode 
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    """
+    input: title, genre, rating
+    output: dictionary with the three keys "title", "genre", and "rating"
+    """
+    movie_dict = {}
+    if title and genre and rating:
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+
+    return None 
+
+
+def add_to_watched(user_data, movie):
+    """
+    input: user_data dictionary with nested list as values, movie dictionary
+    output: updated user_data dictionary 
+    """
+
+    user_data["watched"].append(movie) 
+
+    return user_data
+    
+
+def add_to_watchlist(user_data, movie):
+    """
+    input: user_data dictionary, and movie dictionary
+    output: updated user_data dictionary
+    """
+
+    user_data["watchlist"].append(movie) 
+
+    return user_data

```

Comment:
👍 Nice work! 

Here's some food for thought:

If the movie being passed in isn't valid (missing keys, wrong data type, etc.)? A helper function to sanitize the inputs would be helpful here!